graphql/ui_plugin_build_baron.go

```diff
@@ -52,11 +52,24 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 	settings := env.Settings()
 	queue := env.RemoteQueue()
 	bbProject, ok := BbGetProject(settings, t.Project)
+	var webHook evergreen.WebHook
```

Comment:
This is a big chunk of copied code now (with IsWebhooKConfigured). Can we refactor IsWebhookConfigured to return the webhook so that we can use it here?